src/Tribe/Assets.php

```diff
@@ -77,6 +77,22 @@ public function enqueue_scripts() {
 					],
 				]
 			);
+
+			// @todo: Remove this once we solve the common breakpoints vs container based.
+			tribe_asset(
+				$tickets_main,
+				'tribe-common-responsive',
+				'common-responsive.css',
+				[ 'tribe-common-skeleton-style' ],
+				null,
+				[
+					'groups' => [
+						'tribe-tickets-block-assets',
+						'tribe-tickets-rsvp',
+						'tribe-tickets-registration-page',
+					]
```

Comment:
:no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 3811065f174a5aa6688ef1bda47c39c3</sub>